uint8 test_cmd
---
bool received
